### PR TITLE
PWN-4038 - Added test fee relayer url

### PR DESCRIPTION
**Pull request by 1sakov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-4038

## Description of Work

Added fee relayer change implementation

## Screenshots (Optional)

Screenshot #1 | Screenshot #2 | Screenshot #3
:------------:|:-------------:|:------------:
[Paste Pic 1] | [Paste Pic 2] | [Paste Pic 3]

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/debug/feerelayer/DebugFeeRelayerFragment.kt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import org.p2p.wallet.common.mvp.BaseFragment
+import org.p2p.wallet.databinding.FragmentFeeRelayerBinding
+import org.p2p.wallet.infrastructure.network.environment.EnvironmentManager
+import org.p2p.wallet.utils.popBackStack
+import org.p2p.wallet.utils.viewbinding.viewBinding
+
+class DebugFeeRelayerFragment : BaseFragment(R.layout.fragment_fee_relayer) {
+
+    companion object {
+        fun create() = DebugFeeRelayerFragment()
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/debug/feerelayer/DebugFeeRelayerFragment.kt:17]:
```suggestion
        fun create(): DebugFeeRelayerFragment = DebugFeeRelayerFragment()
```

File: app/src/main/java/org/p2p/wallet/debug/feerelayer/DebugFeeRelayerFragment.kt [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import org.koin.android.ext.android.inject
+import org.p2p.wallet.R
+import org.p2p.wallet.common.AppRestarter
+import org.p2p.wallet.common.mvp.BaseFragment
+import org.p2p.wallet.databinding.FragmentFeeRelayerBinding
+import org.p2p.wallet.infrastructure.network.environment.EnvironmentManager
+import org.p2p.wallet.utils.popBackStack
+import org.p2p.wallet.utils.viewbinding.viewBinding
+
+class DebugFeeRelayerFragment : BaseFragment(R.layout.fragment_fee_relayer) {
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/debug/feerelayer/DebugFeeRelayerFragment.kt:14]: how's `fragment_debug_fee_relayer`?

File: app/src/main/java/org/p2p/wallet/infrastructure/network/NetworkModule.kt [hunk 3]
```diff
@@ -37,9 +39,14 @@ object NetworkModule : InjectionModule {
     const val DEFAULT_READ_TIMEOUT_SECONDS = 30L
 
     override fun create() = module {
-        single { EnvironmentManager(get(), get()) }
+        single { EnvironmentManager(get(), get(), get()) }
         single { TokenKeyProvider(get()) }
         single { CertificateManager(get(), get()) }
+        single {
+            val urlRes = if (BuildConfig.DEBUG) R.string.feeRelayerTestBaseUrl else R.string.feeRelayerBaseUrl
+            val baseUrl = get<Context>().getString(urlRes)
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/infrastructure/network/NetworkModule.kt:47]:
```suggestion
            val baseUrl = androidContext().getString(urlRes)
```

**Review by gslevinkov — APPROVED** [review]

File: app/src/test/java/org/p2p/wallet/swap/koin/CheckModulesTest.kt [hunk 4]
```diff
@@ -99,6 +104,14 @@ class CheckModulesTest : KoinTest {
         every { getExternalFilesDir(any()) }.returns(externalDirMock)
     }
 
+    private val environmentManagerMock: EnvironmentManager = mockk {
```
Comment (gslevinkov) [inline app/src/test/java/org/p2p/wallet/swap/koin/CheckModulesTest.kt:107]: sorry, but it's wrong
you just need to mock shared prefs, no need to create mock for our own class

**Review by gslevinkov — CHANGES_REQUESTED** [review]

**Review by kabramovp2p — APPROVED** [review]

**Review by gslevinkov — APPROVED** [review]

**Pull request merged**